tests/wpunit/PreviewTest.php

```diff
@@ -110,206 +110,529 @@ public function get_query() {
 
 	}
 
+	public function get_query_by_uri() {
+
+		return '
+		query GetPostAndPreview( $id: ID! ) {
+		  post( id: $id idType: URI ) {
+		    ...PostFields
+		    preview {
+		      node {
+		        ...PostFields
+		      }
+		    }
+		  }
+		  preview:post( id: $id idType: URI asPreview: true ) {
+		    ...PostFields
+		  }
+		}
+		fragment PostFields on Post {
+		  __typename
+		  id
+		  title
+		  content
+		  author {
+		    node {
+		      databaseId
+		    }
+		  }
+		  categories {
+		    nodes {
+		      databaseId
+		    }
+		  }
+		  tags {
+		    nodes {
+		      databaseId
+		    }
+		  }
+		  featuredImage {
+		    node {
+		      databaseId
+		    }
+		  }
+		}
+		';
+
+	}
+
+	public function get_query_by_slug() {
```

Comment:
@blakewilson I don't think we need to make this 2 queries. We could just make the `idType` a variable and use the same query with different variable inputs, eh?

Reply:
Doh! Yeah good call. I was in automate/copy/paste mode. I'll take care of that right now.